src/Ledger/Foreign/HSLedger.agda

```diff
@@ -156,11 +156,12 @@ HsGovParams : GovParams
 HsGovParams = record
   { Implementation
   ; ppUpd = let open PParamsDiff in λ where
-      .UpdateT          → ⊤
-      .updateGroups     → λ _ → ∅
-      .applyUpdate      → λ p _ → p
-      .ppdWellFormed    → λ _ → true
-      .ppdWellFormed⇒WF → λ _ _ x → x
+      .UpdateT                → ⊤
+      .updateGroups           → λ _ → ∅
+      .applyUpdate            → λ p _ → p
+      .ppdWellFormed          → λ _ → false
+      .ppdWellFormed⇒hasGroup → λ ()
```

Comment:
Question: I see that Agda fills in the proof (`λ ()`) for us, as the antecedent (`false ≡ true`) is false.  However, I thought we could also prove it explicitly using `⊥-elim`, but somehow that's not working for me.    I would have thought something like `λ f≡t → ⊥-elim f≡t` (which is obviously just the function `⊥-elim`) would work.  The proof `λ ()` works here and I think that means the antecedent is equivalent to `⊥`.  What am I missing?

Reply:
This is simply notation for an empty pattern lambda, so this is almost no magic. If you pattern-match on the proof that `false = true`, `refl` doesn't work so you don't need a body in the pattern lambda.